### PR TITLE
INTDEV-448 separate imports.lp to two files

### PR DESCRIPTION
**Pull request by samutoljamo-pareto**
Separated imports.lp to cardTree.lp and resourceImports.lp. Also updated main.lp to import these new files

### CODE REVIEW

**Comment by samutoljamo-pareto** [comment]
I realized I'll need to also update schema

File: tools/data-handler/src/calculate.ts [hunk 1]
```diff
@@ -193,38 +194,44 @@ export class Calculate {
...
-        return index === -1 ? file : file.substring(0, index);
-      }
-
-      importsContent += `% ${folder}\n`;
-      for (const file of files) {
-        importsContent += `#include "${resolve(join(folder, removeExtension(file) + '.lp')).replace(/\\/g, '/')}".\n`;
-      }
-      importsContent += '\n';
+    // Helper to remove extension from filename.
+    function removeExtension(file: string) {
```
Comment (Fuzzbender) [inline tools/data-handler/src/calculate.ts:202]: Can't you just use `path.parse(file).name`?

File: tools/data-handler/src/calculate.ts [hunk 2]
```diff
@@ -193,38 +194,44 @@ export class Calculate {
...
 
+    importsContent += `% ${folder}\n`;
+    for (const file of files) {
+      importsContent += `#include "${resolve(join(folder, removeExtension(file) + '.lp')).replace(/\\/g, '/')}".\n`;
+    }
+    importsContent += '\n';
     await writeFileSafe(destinationFile, importsContent);
   }
 
+  private async generateResourceImports() {
```
Comment (Fuzzbender) [inline tools/data-handler/src/calculate.ts:215]: Put these `private` generate-methods in alphabetical order.

**Review by Fuzzbender — APPROVED** [review]

File: tools/data-handler/src/calculate.ts (hunk 1 above)
Comment (samutoljamo-pareto) [reply tools/data-handler/src/calculate.ts:202]: Good catch

**Pull request merged**